front/app/components/Form/Components/Layouts/PageControlButtons.tsx

```diff
@@ -50,10 +55,36 @@ const PageControlButtons = ({
   hasPreviousPage,
   isLoading,
   pageVariant,
+  phases,
+  currentPhase,
 }: Props) => {
   const theme = useTheme();
   const isSmallerThanPhone = useBreakpoint('phone');
 
+  const getButtonText = () => {
```

Comment:
very minor, but might be better named `getButtonMessage` given that it returns a message descriptor